src/plugins/home/server/services/sample_data/data_sets/logs/saved_objects.ts

```diff
@@ -398,7 +398,7 @@ export const getSavedObjects = (): SavedObject[] => [
       description: '',
       kibanaSavedObjectMeta: {
         searchSourceJSON:
-          '{"query":{"query":"","language":"kuery"},"filter":[],"indexRefName":"kibanaSavedObjectMeta.searchSourceJSON.index"}',
+          '{"index":"90943e30-9a47-11e8-b64d-95841ca0b247","query":{"query":"","language":"kuery"},"filter":[]}',
```

Comment:
why do we change from indexRefName":"kibanaSavedObjectMeta.searchSourceJSON.index to doc id?

Reply:
so if you check the other[ sample visualizations](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/58d49ca0d160814201712dfbf5f13282caa82db7/src/plugins/home/server/services/sample_data/data_sets/logs/saved_objects.ts#L140) that are working fine, they all following same pattern
1. define "index" in "searchSourceJson"
2. define "references" field as empty array.

But visualizations in this file are not following the pattern, that's why only them will fail our logic to add reference to data source, and fail to render with data source.

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/58d49ca0d160814201712dfbf5f13282caa82db7/src/plugins/home/server/services/sample_data/data_sets/util.ts#L43